### DISCUSSION TITLE
MediaElement album name, song name and album art

### DISCUSSION THREAD

**Discussion by adumitru2019** [CommunityToolkit/Maui#1765] [New Feature Discussions]
I suggest adding a new feature to the Community Toolkit Media Element to show the album name, the song name and the album art when playing an audio source (from local or URI source).

**Comment by ne0rrmatrix** [comment]
Working on it here : https://github.com/CommunityToolkit/Maui/pull/1782 I hope it will be reviewed soon and the team decides whether to go forward with my PR.

**Comment by TheCodeTraveler** [comment]
Added in https://github.com/CommunityToolkit/Maui/pull/1782
